### DISCUSSION TITLE
Container can't access other services through their public URL

### DISCUSSION THREAD

**Discussion by Fry98** [coollabsio/coolify#2887] [Q&A]
In my testing, when a service tries to access other services (or itself for that matter) through their external URL, the request keeps pending and never gets a response. In my specific use case, I want my frontend server (let's say on myapp.com) to be able to fetch data from the API (on api.myapp.com) during SSR. In this example, both services and Coolify itself are all deployed on the same server. Is there a way to make this work?

**Comment by snimshchikov** [comment]
This is a docker problem (if containers are using the same network, and all coolify containers are using the same network). 

You can enable `project->advanced->Container Names->Consistent Container Names` and set container id instead of domain.
